### PR TITLE
Clean dockerfiles

### PR DESCRIPTION
**Pull request by aslafy-z**
- Rebase on `nginx:1-alpine`
- Delete deprecated maintainer fields
- Delete `bash` deps as it's not needed
- Use COPY instead of ADD
- Lint shellscript

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,7 @@
-FROM alpine:edge
-MAINTAINER José Moreira <josemoreiravarzim@gmail.com>
```
Comment (cusspvz) [inline Dockerfile]: @KyleAMathews @DSchau can we create an alias/group email for the gatsby's maintainers so we can expose it here?

**Comment by aslafy-z** [comment]
For reference: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

**Comment by cusspvz** [comment]
> For reference: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

I knew, but thanks for bringing that up. I just find useful to have a centralized maintainer's email instead of placing our own ones there.

**Comment by aslafy-z** [comment]
@cusspvz sure, I shared it just to point out the use of `LABEL maintainer=".."` instead of the `MAINTAINER` instruction

File: Dockerfile (hunk 1 above)
Comment (aslafy-z) [reply Dockerfile]: It will also have to go in #19

**Pull request closed without merging**